api/openapi-spec.yaml

```diff
@@ -0,0 +1,409 @@
+openapi: 3.0.1
+info:
+  title: Event Repository REST API
+  description: Event Repository REST API for retrieving the event information
+  contact: {}
+  version: 0.0.1
+tags:
+- name: event-resource
+  description: The Event Resource API for getting single event information
+- name: search-resource
+  description: The search api provides methods for querying up/down stream given an
+    event id.
+- name: events-resource
+  description: The Events Resource API for getting all events information
+paths:
+  /events:
+    get:
+      tags:
+      - events-resource
+      summary: To get all events information
+      operationId: getEventsUsingGET
```

Comment:
Given that operationId sometimes is used to generate functions in client libraries I think we should choose a name that would work well in that scenario, e.g getEvent, getEvents, and getLinkedEvents for the three operations currently covered by the API spec.